### PR TITLE
Added missing Color operators

### PR DESCRIPTION
**Pull request by Zylann**
Fixes https://github.com/GodotNativeTools/godot-cpp/issues/405

### CODE REVIEW

File: include/core/Color.hpp [hunk 1]
```diff
@@ -73,6 +74,23 @@ struct Color {
...
+
+    Color operator*(const Color &p_color) const;
+    Color operator*(const real_t &rvalue) const;
+    void operator*=(const Color &p_color);
+    void operator*=(const real_t &rvalue);
+
+    Color operator/(const Color &p_color) const;
+    Color operator/(const real_t &rvalue) const;
+    void operator/=(const Color &p_color);
+    void operator/=(const real_t &rvalue);
```
Comment (karroffel) [inline include/core/Color.hpp:92]: Indentation here seems to be different than all other lines

Comment (Zylann) [reply include/core/Color.hpp:92]: It's a big copy/paste from Godot's codebase, and I use clang-format... strange it didnt get picked up

**Comment by Zylann** [comment]
Updated formatting, however build fails on windows due to some shell command failing?

**Comment by karroffel** [comment]
> Updated formatting, however CI fails on windows due to some shell command failing?

It seems like some download is failing, not related to the PR. I can't seem to be able to restart the CI job, maybe you can and if not a force-push should re-trigger CI.

**Comment by Zylann** [comment]
Worked this time

**Comment by BastiaanOlij** [comment]
Merged this now it passed travis tests, looks fine to me, thanks @Zylann

**Pull request merged**